spec/integration/profiles_spec.rb

```diff
@@ -255,7 +255,7 @@
                        type: { type: :string },
                        id: ref_schema('uuid'),
                        attributes: ref_schema('benchmark'),
-                       relationships: ref_schema('benchmark_relationships')
+                       relationships: {}
```

Comment:
This is required to be a schema. An always-empty object would look like this:

```suggestion
                       relationships: { type: :object, properties: {} }
```